src/js/embeds.js

```diff
@@ -298,6 +298,10 @@
             success: function(data) {
                 var html = data && data.html;
 
+		if (that.options.storeMeta){
+                    html+='<div class="medium-insert-meta"><script type="text/json">'+JSON.stringify(data)+'</script></div>';                    
```

Comment:
Could you use `medium-insert-embeds-meta` class instead?
And don't forget to add space before and after `+=`